start.bat

```diff
@@ -111,3 +102,19 @@ set CLASSPATH=%TOOLS%;%RSYNTAXTEXTAREA%;%GSON%;%BYTEMAN%;%JRD%;%CPLC%
 if "%PORTABLE_JRD_HOME:~0,2%"=="//" (
   popd
 )
+
+rem Exit before function section
+exit /B 0
+
+rem Function for finding mvn repo files (2nd arg) of a certain group (1st arg) & returning it through the 3rd arg
+:findLib
```

Comment:
This works? You can really ahve function declaration after usage in bats?

Reply:
To answer your first question, yes it works. Both in development and in a portable image.